src/Analyzers/CSharp/Tests/RemoveUnnecessarySuppressions/RemoveUnnecessarySuppressionsTests.cs

```diff
@@ -182,5 +183,23 @@ class C
 }}";
             await VerifyCS.VerifyCodeFixAsync(input, input);
         }
+
+        [Fact, WorkItem(45465, "https://github.com/dotnet/roslyn/issues/45465")]
+        public async Task LegacyModeGlobalSuppressionWithNamespaceAndDescendantsScope()
+        {
+            var target = "N:N.InvalidChild";
+            var input = $@"
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(""Category"", ""Id: Title"", Scope = ""namespaceanddescendants"", Target = {{|#0:""{target}""|}})]
```

Comment:
❔ Do we have any tests with invalid `Scope`?

Reply:
Yep - https://github.com/dotnet/roslyn/blob/be56bf44b7bcd2462c123aca88282791b4e5bca0/src/Analyzers/CSharp/Tests/RemoveUnnecessarySuppressions/RemoveUnnecessaryAttributeSuppressionsTests.cs#L81